yarn.lock

```diff
@@ -15884,7 +15893,7 @@ next-tick@~1.0.0:
   resolved "https://registry.yarnpkg.com/next-tick/-/next-tick-1.0.0.tgz#ca86d1fe8828169b0120208e3dc8424b9db8342c"
   integrity sha1-yobR/ogoFpsBICCOPchCS524NCw=
 
-next@12, next@^11.0.1:
+next@^11.0.1, next@latest:
   version "11.1.2"
```

Comment:
Looks like the resolved version is still pointing to 11.1.2. Should we update https://github.com/aws-amplify/amplify-ui/blob/fe5f2d88cba29b5eff8ca99df8c10c7b1c5f9e1b/package.json#L39 as well?

And if we go from 11 to 12, do we need to make adjustment to Amplify Console to reflect the update? I think this is more of a question to @ErikCH or @ericclemmons who has worked on hosting on console side.

Reply:
My understanding was that we had already upgraded to 12 in https://github.com/aws-amplify/amplify-ui/pull/895. I don't think anything is currently using 11, so I updated this to `^12.0.7`. Let me know if I'm still misunderstanding. 

It may also makes sense to remove this resolutions entry, as I don't know what it's used for anymore. I'm guessing it was necessary before when we had to match the main docs site next js version. @ericclemmons @wlee221 thoughts?

Reply:
Hmm, but on `main` I'm seeing  `"version": "11.1.2",` in node_modules. Is that because of this resolutions entry?

Reply:
Yeah, I think it makes sense to remove that resolution entry in favor of explicit dependencies on `docs/` and `examples/next`.